### PR TITLE
include helm and pipelines in +Add page

### PR DESCRIPTION
**Pull request by christianvogt**
**Analysis / Root cause**: 
Request to include operator backed, helm and pipeline cards in +Add page for discovery.

**Solution Description**: 
Use extension mechanism to include new cards in the +Add page.
Updated event sources icon as well

**Screen shots / Gifs for design review**: 

image

cc @openshift/team-devconsole-ux @siamaksade 

**Test setup:**
Helm card will always be present.
Pipeline card will only appear if pipeline operator is installed.

**Browser conformance**: 

- [ ] Chrome
- [ ] Firefox
- [x] Safari
- [x] Edge

### CODE REVIEW

File: frontend/packages/dev-console/src/images/pipeline.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...26.04 64.66 31.49"/><path class="cls-3" d="M27.71,66.19a27.56,27.56,0,0,1,1.82-34.64"/><polygon class="cls-4" points="31.49 35.34 33.81 27.71 26.04 29.52 31.49 35.34"/><path class="cls-3" d="M66.19,72.29a27.56,27.56,0,0,1-34.64-1.82"/><polygon class="cls-4" points="35.34 68.51 27.71 66.19 29.52 73.96 35.34 68.51"/><path class="cls-3" d="M72.29,33.81a27.56,27.56,0,0,1-1.82,34.64"/><polygon class="cls-4" points="68.51 64.66 66.19 72.29 73.96 70.48 68.51 64.66"/><path class="cls-4" d="M44.86,27.93a7.28,7.28,0,1,1,10.31,0,7.16,7.16,0,0,1-10.31,0"/><path class="cls-5" d="M44.86,27.93,55.15,17.64a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.31,0"/><path class="cls-4" d="M44.86,82.66a7.28,7.28,0,1,1,10.31,0,7.17,7.17,0,0,1-10.31,0"/><path class="cls-5" d="M44.86,82.66,55.15,72.37a7.33,7.33,0,0,1,0,10.31,7.17,7.17,0,0,1-10.31,0"/><path class="cls-4" d="M17.31,55.11a7.27,7.27,0,1,1,10.3,0,7.16,7.16,0,0,1-10.3,0"/><path class="cls-5" d="M17.31,55.11,27.59,44.82a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.3,0"/><path class="cls-4" d="M72.41,55.11a7.27,7.27,0,1,1,10.3,0,7.16,7.16,0,0,1-10.3,0"/><path class="cls-5" d="M72.41,55.11,82.69,44.82a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.3,0"/></svg>
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/images/pipeline.svg:1]: This is a UX approved icon?

File: frontend/packages/dev-console/src/components/EmptyState.tsx [hunk 2]
```diff
@@ -38,8 +38,9 @@ const Item: React.FC<ItemProps> = ({
         onClick={(e: React.SyntheticEvent) => navigateTo(e, resolvedUrl)}
         href={resolvedUrl}
         title={label}
-        iconImg={icon}
+        iconImg={typeof icon === 'string' ? icon : undefined}
         iconClass={iconClass}
+        icon={typeof icon === 'object' ? icon : undefined}
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/EmptyState.tsx:43]:
```suggestion
        icon={React.isValidElement(icon) ? icon : undefined}
```
Pretty sure this is the check you want.

File: frontend/packages/dev-console/src/images/pipeline.svg (hunk 1 above)
Comment (serenamarie125) [reply frontend/packages/dev-console/src/images/pipeline.svg:1]: We're working on getting another SVG, but it's very similar yes :) It's the b&w version of the operator logo

File: frontend/packages/dev-console/src/components/EmptyState.tsx (hunk 2 above)
Comment (christianvogt) [reply frontend/packages/dev-console/src/components/EmptyState.tsx:43]: Indeed i do!

Comment (christianvogt) [reply frontend/packages/dev-console/src/components/EmptyState.tsx]: fixed. thanks @andrewballantyne

**Comment by spadgett** [comment]
/hold
for #5100

**Pull request merged**